ingestion/src/metadata/ingestion/source/database/common_db_source.py

```diff
@@ -260,7 +260,11 @@ def yield_database_schema(
         )
         source_url = (
             SourceUrl(source_url)
-            if (source_url := self.get_source_url(database_name=schema_name))
+            if (
+                source_url := self.get_source_url(
+                    database_name=self.context.get().database, schema_name=schema_name
+                )
+            )
```

Comment:
lets make sure that `database_name=schema_name` was not intentional for any specific database